reqs/requirements-dev-lite.txt

```diff
@@ -15,5 +15,6 @@ pytest-order>=0.9.5
 pytest-random-order>=1.0.4
 pytest-timeout>=2.1.0
 requirements-parser>=0.2.0
+responses>=0.23.1 # requests mocking
```

Comment:
If we go this route, can we make some noise so the team knows to use this moving forward?